libmscore/measure.h

```diff
@@ -210,6 +210,7 @@ class Measure final : public MeasureBase {
       AccidentalVal findAccidental(Segment* s, int staffIdx, int line, bool &error) const;
       void exchangeVoice(int voice1, int voice2, int staffIdx);
       void checkMultiVoices(int staffIdx);
+      bool hasVoices(int staffIdx, Fraction stick, Fraction len);
```

Comment:
Um, two `hasVoices()` (with different types of parameters) not adjacent (the other one is in [line 119](https://github.com/musescore/MuseScore/blob/478197086fdd3ae1cf2dc1ff33d037b5bfb86714/libmscore/measure.h#L119))? I know that this version is related to `checkMultiVoices()` but maybe some adjustments can be made to put the two together?

Reply:
Okay, I moved this line to be adjacent to the declaration of the other `hasVoices()`. I also found a few more places to call the new `hasVoices()` rather than the old one.